spynnaker/pyNN/models/recording_common.py

```diff
@@ -20,9 +20,11 @@ def __init__(self, population):
         """ object to hold recording behaviour
 
         :param population: the population to record for
+        :param simulator: the spinnaker control class
```

Comment:
I don't think this has actually been added as a parameter...

Reply:
it had been and the replaced with cleaner getting via global variables